SYCL/Plugin/interop-level-zero-thread.cpp

```diff
@@ -168,6 +168,7 @@ sycl::event operation(sycl::queue q) {
       {l0_event, sycl::ext::oneapi::level_zero::ownership::keep},
       q.get_context());
 
+  assert(zeEventHostSignal(l0_event) == 0);
```

Comment:
why is the signal added?
also, it will not be executed if asserts are disabled

Reply:
There are two barriers submitted one after the other with the output event of the first as an input event of the second. The second barrier takes an additional input event, which is newly created, and that no one signals. This causes a hang with immediate commandlists. I found that signaling the newly created event which is input to the second barrier eliminates the hang. I'll remove the assert (copied from other parts of the code) and just signal it.

Reply:
Hmm, I wonder how it works with batching then

Reply:
I don't fully understand it, but I think the situation is unique to immediate command list usage in the Driver.

Reply:
OK, let it be